docs/design/eth-filter.md

```diff
@@ -0,0 +1,201 @@
+    "id": 73,
+    "result": [{
+        "address": "0xb5a5f22694352c15b00323844ad545abb2b11028",
+        "blockHash": "0x99e8663c7b6d8bba3c7627a17d774238eae3e793dee30008debb2699666657de",
+        "blockNumber": "0x5d12ab",
+        "data": "0x0000000000000000000000000000000000000000000000a247d7a2955b61d000",
+        "logIndex": "0x0",
+        "removed": false,
+        "topics": ["0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef", "0x000000000000000000000000bdc0afe57b8e9468aa95396da2ab2063e595f37e", "0x0000000000000000000000007503e090dc2b64a88f034fb45e247cbd82b8741e"],
+        "transactionHash": "0xa74c2432c9cf7dbb875a385a2411fd8f13ca9ec12216864b1a1ead3c99de99cd",
+        "transactionIndex": "0x3"
+    }]
+}
+```
+
+### Getting Filter Changes
+
+     The `getFilterChanges` method returns an array of logs which occurred since last poll.
+
+#### Parameters
+
+An existing `FILTER_ID` in hex format
+
+#### Request
+
+```javascript
+
+{
+  "id": 1,
+  "jsonrpc": "2.0",
+  "method": "eth_getFilterChanges",
+  "params": [
+    "0x62440eb3b951769ef7cc8abb1d26fbaa"
+  ]
+}
+
+```
+
+#### Response
+
+```javascript
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "result": [
+    {
+      "address": "0xe41d2489571d322189246dafa5ebde1f4699f498",
+      "blockHash": "0x8243343df08b9751f5ca0c5f8c9c0460d8a9b6351066fae0acbd4d3e776de8bb",
+      "blockNumber": "0x429d3b",
+      "data": "0x00000000000000000000000000000000000000000000006194049f30f7200000",
+      "logIndex": "0x1",
+      "removed": false,
+      "topics": [
+        "0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef",
+        "0x000000000000000000000000e4c8eb504eeeffb8a0468318a96d565d7521aef3",
+        "0x0000000000000000000000004a7a5c1f34c57b9d1e0993e83060b6736f6a42bd"
+      ],
+      "transactionHash": "0x78356eec0d6ed3087e277538811f604c329be8217c5b5e007e4eeb3dba973bff",
+      "transactionIndex": "0x2"
+    }
+  ]
+}
+```
+
+## Error Codes
+
+| Error Codes |        Error message        |                              Solution                               |
+| :---------: | :-------------------------: | :-----------------------------------------------------------------: |
+|    32000    |      Filter not found.      | Occurs when user attempts receive logs using non-existing filterID. |
+|    32602    | Log response size exceeded. |          Occurs when user request exceed logs size limit.           |
+
+## Limits
+
+1. All filters expire after 5 minutes of inactivity (no queries). Env. variable can be `FILTER_CACHE_TTL`.
+2. Returned logs should have limitations, similar to those used in `eth_getLogs` method.
+
+## Metric Capturing
+
+Capture metrics for the following:
+
+1. Log every call to all filter API method, as a total amount.
+2. The number of active filters.
+3. The duration of active filters.
+4. The total amount of requests to the mirror node per filter.
+5. The amount of requests to the mirror node per filter that have returned non-null data.
+
```

Comment:
6. Amount of filters that were uninstalled by Time Limit vs Explicitly by the user.

Reply:
Thanks, added.